internal/user_access.go

```diff
@@ -65,6 +65,15 @@ func AddNewUser(newUserProfile model.LasagnaLoveUser) (model.LasagnaLoveUser, er
 		return model.LasagnaLoveUser{}, errors.New("one or more required fields missing or empty")
 	}
 
+	
```

Comment:
Not a blocking issue, but it'd be ideal to reduce the whitespace between sections of code to one line. It makes it easier for others to read code when there's more that can fit on one screen at a time.